lib/tapjoy/autoscaling_bootstrap/AWS/ec2.rb

```diff
@@ -55,6 +60,29 @@ def request_spot_fleet(spot_price:, target_capacity:,
                 allocation_strategy: allocation_strategy
               })
           end
+
+          def set_recipes_and_tags(user_data_config, launch_config)
+            # Look for specified recipes and tags in the launch and overall spot fleet config
+            # and set the appropriate values
+            case
+            when launch_config[:recipes]
```

Comment:
Please don't call it launch_config. =)  It took me a while to figure out that this had nothing to do with ASG related stuff